test/unit/services/autocomplete.ts

```diff
@@ -70,11 +70,21 @@ suite('Autocomplete Service', ({ expect, spy, stub, itShouldBeCore, itShouldExte
 
   describe('registerAutocomplete()', () => {
     it('should add tag to registeredAutocompleteTags', () => {
-      const tag: any = { a: 'b' };
+      let unmountCb;
+      const on = (e, cb) => {
```

Comment:
You don't need to do all of this, you can do 
```
on = spy();
...
on.getCall(0).args[1]
```
in order to get the function passed into on.